server_development/topics/providers.adoc

```diff
@@ -527,6 +527,78 @@ ifeval::["{kc_dist}" == "wildfly"]
 Once you have a JAR file with a descriptor and the scripts you want to deploy, you just need to copy the JAR to the {project_name} `standalone/deployments/` directory.
 endif::[]
 
+===== Deploy the script engine on Java 15 and later
+
+In order to run the scripts, JavaScript providers require JavaScript engine to be available in your Java application. In the Java 14 or earlier, there is Nashorn JavaScript Engine
+automatically available as part of the Java itself and JavaScript providers are able to leverage this script engine out of the box. However in case of Java 15 or later, script engine is not part
+of the Java itself and needs to be added to your server. {project_name} does not have any script engine by default. So in case of Java 15 or later, there is additional step needed when deploying script
```

Comment:
```suggestion
To run the scripts, JavaScript providers require that a JavaScript engine is available in your Java application. Java 14 and lower versions include the Nashorn JavaScript Engine. It is 
automatically available as part of the Java itself and JavaScript providers are able to use this script engine by default. However, for Java 15 or higher versions, the script engine is not part
of the Java itself. It needs to be added to your server because {project_name} does not have any script engine by default. Java 15 and higher versions require an extra step when deploying script
```